src/model/MazePoint.java

```diff
@@ -0,0 +1,48 @@
+package model;
+
+public class MazePoint {
+
+    private int x;
+    private int y;
+    private MazePoint parent;
```

Comment:
Maze point is only used when solving the maze, right? (BFS?)

Reply:
Nope, as we discussed, I decided to create a custom mazePoint class instead of using just a Point (double irrelevant parameters inside this old class).